packages/@tinacms/toolkit/src/packages/fields/plugins/GroupListFieldPlugin.tsx

```diff
@@ -83,29 +84,40 @@ const Group = ({ tinaForm, form, field, input }: GroupProps) => {
     [field.itemProps]
   )
 
+  // @ts-ignore
+  const isMax = items.length >= (field.max || Infinity)
+  // @ts-ignore
+  const isMin = items.length <= (field.min || 0)
+  // @ts-ignore
+  const fixedLength = field.min === field.max
```

Comment:
Do we need this check?

Reply:
It's used to determine if add/remove buttons should be rendered. If they're not the same the UI should only use a disabled state to indicate it can't currently be added/removed.

Reply:
Here's a field with `min: 3, max: 3` where the add/remove UI isn't needed at all:

<img width="401" alt="Screenshot 2023-01-11 at 1 39 08 PM" src="https://user-images.githubusercontent.com/5075484/211878455-455eb8d5-a463-42c0-86d9-74452eb6cfbc.png">